drive:
    ros__parameters:
        visualization: true
        model_path: "/lanefollowing/ros2_ws/src/lane_following/model/model.h5"
        camera_topic: "/simulator/sensor/camera/center/compressed"
        control_topic: "/lanefollowing/steering_cmd"
        steer_ratio: 16.
        steering_wheel_single_direction_max: 470.
        wheel_base: 2.836747
